### PR TITLE
feat(editor/laterbinding): Added first part for SMV Later Binding Editor

### LINKED ISSUE #869: Add Subscriber Later Binding (SMV) plugin with filterable FCDA list
As a user of OpenSCD I want to subscriber data from GOOSE control blocks `GSEControl` to existing inputs `ExtRef`. 

**Requirements**
- Allow filtering for `FCDA` elements with the search string
- `IED` and `GSEControl` shall be `noninteractive`
- show `GSEControl` name as primary and its identity string as secondary slot
- for `FCDA` element primary slot is `doName.daName` and secondary slot `ldInst/prefix lnClass lnInst`
**Mockup**

### LINKED ISSUE #870: Subscriber Later Binding (SMV): Add edit button to open `SampledValueControl` element
As a user of OpenSCD I want to be able to change the `SampledValueControl` element from within the `Subscriber Later Binding (SMV)` plugin.

**Requirement:**
- add edit button to the `SampledValueControl` list item that opens the edit wizard for `SampledValueControl` element

### PR DESCRIPTION
**Pull request by dlabordus**
closes #869 and closes #870

### CODE REVIEW

**Comment by JakobVogelsang** [comment]
I did the tests in terms of clicking and UI and have two remarks there:
1. I saw that you have added the IED name to the list items of the `FCDA`, but the control block name is missing. That means that users typing in the name of the `SampledValueControl` element will see only the noninteractive list items and not their children. I would suggest adding the complete identity string of the `SampledValueControl` element to the list item value representing the `FCDA` as well. 
2. On my machine, there are two scroll bars. That is a bit irritating

File: src/editors/SMVSubscriberLaterBinding.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      </div>
+    </div>`;
+  }
+
+  static styles = css`
+    :host {
+      width: 100vw;
+    }
+
+    .container {
```
Comment (JakobVogelsang) [inline src/editors/SMVSubscriberLaterBinding.ts:26]: I would remove the next two rules for now. In this PR they are not needed, and then it makes more trouble than good

File: src/editors/subscription/smv-laterbinding/svc-laterbinding-list.ts [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    const wizard = wizards['SampledValueControl'].edit(svcElement);
+    if (wizard) this.dispatchEvent(newWizardEvent(wizard));
+  }
+
+  renderFCDA(fcdaElement: Element): TemplateResult {
+    return html`<mwc-list-item
+      graphic="large"
+      twoline
+      class="subitem"
+      value="${identity(fcdaElement)}"
```
Comment (JakobVogelsang) [inline src/editors/subscription/smv-laterbinding/svc-laterbinding-list.ts:64]: This should be joined with the identity string of the referenced `SampledValueControl` element. Then filtering on the `SampledValueControl` element does never hide its children.

File: src/translations/de.ts [hunk 3]
```diff
@@ -366,6 +366,11 @@ export const de: Translations = {
           'Zeigt verbundene Sampled Value(s) des ausgewählten IED',
       },
     },
+    smvLaterBinding: {
+      svcList: {
+        noSvcFound: '????',
```
Comment (JakobVogelsang) [inline src/translations/de.ts:371]:
```suggestion
        noSvcFound: 'Kein Sampled-Value-Kontrolblock im Projekt',
```

**Review by JakobVogelsang — CHANGES_REQUESTED** [review]
Three changes so far I see :)

File: src/editors/subscription/smv-laterbinding/svc-laterbinding-list.ts (hunk 2 above)
Comment (dlabordus) [reply src/editors/subscription/smv-laterbinding/svc-laterbinding-list.ts:64]: Fixed

File: src/translations/de.ts (hunk 3 above)
Comment (dlabordus) [reply src/translations/de.ts:371]: Updated

File: src/editors/SMVSubscriberLaterBinding.ts (hunk 1 above)
Comment (dlabordus) [reply src/editors/SMVSubscriberLaterBinding.ts:26]: Fixed

**Review by JakobVogelsang — APPROVED** [review]
LGTM

**Pull request merged**